hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java

```diff
@@ -142,7 +142,7 @@ public Character readObject(GenericRecord genericRecord, String fieldName) {
 
             @Override
             public void writeFieldFromRecordToWriter(DefaultCompactWriter writer, GenericRecord genericRecord, String fieldName) {
-                writer.writeChar(fieldName, genericRecord.getChar(fieldName));
```

Comment:
Instead of unsupported exception, can we completely remove the operation itself for CHAR and CHAR_ARRAY from the list of operations?

Reply:
No we can't, as `writeJsonFormattedField` and `readObject` are used for Portable as well